tests/test_access/tst_access_drogon_manage_login.py

```diff
@@ -101,7 +101,10 @@ def test_aggregations_bulk(explorer: Explorer):
     cases = explorer.cases
     print("Number of cases: ", len(cases))
     assert len(cases) > 0
-    case = cases[0]
+    for c in cases:
+        if len(c.iterations) > 0 and len(c.surfaces) > 40:
+            case = c
+            break
```

Comment:
@adnejacobsen @rwiker 
Is the Explorer able to do this filtering for us?

Reply:
I assume no answer means no such support